alien/CMakeLists.txt

```diff
@@ -56,18 +56,33 @@ arcane_framework_set_if_unset(Arctools_ROOT ${ARCFRAMEWORK_ROOT}/arctools PATH "
 
 list(APPEND CMAKE_MODULE_PATH ${ARCFRAMEWORK_ROOT}/_common/build_all/cmake)
 
-# Add Alien only if blas and boost found
+#first try to look for MKL
+message(STATUS "Looking for mkl BLAS")
+set(BLA_VENDOR Intel10_64lp_seq)
```

Comment:
Platform specific, should not be hard coded in generic CMake file.

Reply:
I agree but cmake find_package(BLAS) does give information on which vendor was found. It also does not look for include dir.

Reply:
BLAS support is awful, I totally agree. Nevertheless, Intel10 + sequential by default feels quite a bit strange.